README.md

```diff
@@ -79,22 +79,23 @@ This will create your docker containers and initialize the database by running m
    `python -m pip install wheel`
    1. Install our requirements:
    `python -m pip install -r requirements.txt`
-   1. Initialize pre-commit
-   `pre-commit install`
+7. Initialize pre-commit:
+   * Using venv: `pre-commit install`
+   * Using docker: `docker-compose run web bash -c "pre-commit install"`
```

Comment:
> I'm confused by this. Are you committing within docker?

Yes. Since I don't have a Python virtual environment setup, I am unable to run the test suite outside of docker. This means that I am also unable to run the pre-commit hooks outside of docker. For this reason, whenever I run `git commit`, I am required to run it inside of docker. It's a slightly annoying workaround, but it's been working fine for me 😄.